modules/rostests/winetests/mshtml/activex.c

```diff
@@ -2434,6 +2434,7 @@ static void test_flash_ax(void)
         skip("Skipping test_ui_activate(). ROSTESTS-114.\n");
         skip("Skipping test_container(notif_doc). ROSTESTS-114.\n");
         skip("Skipping test_object_elem(notif_doc). ROSTESTS-114.\n");
+        return;
```

Comment:
Not sure I fully understand what's going on. Do you need the return, because the rest of the tests below depend on what has been done in those skipped "test_ui_activate()" etc. calls?

Reply:
it will simply crash below, if these tests don't run.

Reply:
ok so there's a dependency on the skipped tests.